Detections/Tenable/Tenable - Med Vulnerability found.yaml

```diff
@@ -0,0 +1,20 @@
+id: 9d7b2b02-8b44-4850-89e9-a9ca442bc834
+name: Tenable - Medium Vulnerability found
+description: |
+  'This query identifies a Tenable Vulnerability of Medium Vulnerability.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Tenable
+    dataTypes: 
+      - Tenable_Assets_List
+queryFrequency: 12h
+queryPeriod: 12h
+triggerOperator: gt
+triggerThreshold: 0
+query: |
+
+  Tenable_Assets_List
+  | join (Tenable_Vulns_List) 
```

Comment:
Based on your custom table, this needs to be Tenable_Vulns_List_CL

Reply:
Hey @shainw I was assuming the parsers were imported, would you rather use _CL in the detections?

Reply:
Bump @shainw for thoughts on above